gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/Policy.java

```diff
@@ -0,0 +1,389 @@
+
+      Builder() {
+        members = new ArrayList<Member>();
+      }
+
+      public Builder role(RoleType role) {
+        this.role = role;
+        return this;
+      }
+
+      public Builder members(List<Member> members) {
+        this.members = checkNotNull(members);
+        return this;
+      }
+
+      public Builder clearMembers() {
+        this.members = new ArrayList<>();
+        return this;
+      }
+
+      public Builder addMember(Member member) {
+        this.members.add(member);
+        return this;
+      }
+
+      public Builder removeMember(Member member) {
+        this.members.remove(member);
+        return this;
+      }
+
+      public Binding build() {
+        return new Binding(role, members);
+      }
+    }
+
+    private Binding(RoleType role, List<Member> members) {
+      this.role = role;
+      ImmutableList.Builder<Member> membersListBuilder = new ImmutableList.Builder<>();
+      for (Member member : members) {
+        membersListBuilder.add(member);
+      }
+      this.members = membersListBuilder.build();
+    }
+
+    public static Binding binding(RoleType role, List<Member> members) {
+      return new Binding(role, members);
+    }
+
+    public RoleType role() {
+      return role;
+    }
+
+    public List<Member> members() {
+      return members;
+    }
+
+    public static Builder builder() {
+      return new Builder();
+    }
+
+    public Builder toBuilder() {
+      List<Member> mutableMembers = new ArrayList<>();
+      for (Member member : members) {
+        mutableMembers.add(member);
+      }
+      return new Builder().role(role).members(mutableMembers);
+    }
+
+    com.google.api.services.cloudresourcemanager.model.Binding toPb() {
+      com.google.api.services.cloudresourcemanager.model.Binding bindingPb =
+          new com.google.api.services.cloudresourcemanager.model.Binding();
+      if (role != null) {
+        bindingPb.setRole("roles/" + role.toString().toLowerCase());
+      }
+      List<String> membersPb = new ArrayList<>(members.size());
+      for (Member member : members) {
+        if (member.emailOrDomain() != null) {
+          membersPb.add(member.type().prefix() + member.emailOrDomain());
+        } else {
+          membersPb.add(member.type().prefix());
+        }
+      }
+      bindingPb.setMembers(membersPb);
+      return bindingPb;
+    }
+
+    static Binding fromPb(com.google.api.services.cloudresourcemanager.model.Binding bindingPb) {
+      RoleType role =
+          (bindingPb.getRole() == null)
+              ? null : RoleType.valueOf(bindingPb.getRole().split("/")[1].toUpperCase());
+      List<Member> members = new ArrayList<>();
+      if (bindingPb.getMembers() != null) {
+        for (String memberPb : bindingPb.getMembers()) {
+          String[] memberInfo = memberPb.split(":", 2);
+          String memberTypeStr = memberInfo[0];
+          String emailOrDomain = (memberInfo.length > 1) ? emailOrDomain = memberInfo[1] : null;
```

Comment:
Simply `String emailOrDomain = (memberInfo.length > 1) ? memberInfo[1] : null;` should do it